pydicom/tag.py

```diff
@@ -1,20 +1,36 @@
-"""Define Tag class to hold a DICOM (group, element) tag."""
-# Copyright (c) 2008-2012 Darcy Mason
-# This file is part of pydicom, released under a modified MIT license.
-#    See the file LICENSE included with this distribution, also
-#    available at https://github.com/pydicom/pydicom
-
-# Store the 4 bytes of a dicom tag as an arbitary length integer
-#      (python "long" in python <3; "int" for python >=3).
-# NOTE: This must be not be stored as a tuple internally, as some code logic
-#       (e.g. in write_AT of filewriter) checks if a value is a multi-value
+# Copyright 2008-2017 pydicom authors. See LICENSE file for details.
+"""Define Tag class to hold a DICOM (group, element) tag and related functions.
+
+The 4 bytes of the DICOM tag are stored as an arbitrary length 'int'. Tags are
+stored as a single number and separated to (group, element) as required.
+"""
```

Comment:
better leave the `integer` part as in the previous description - it is still long in Python 2